docs/installing.rst

```diff
@@ -108,13 +110,10 @@ On Windows [5]_:
 Python and OS Compatibility
 ---------------------------
 
-pip works with CPython versions 2.6, 2.7, 3.3, 3.4, 3.5 and also pypy.
-
-This means pip works on the latest patch version of each of these minor versions
-(i.e. 2.6.9 for 2.6, etc).
-Previous patch versions are supported on a best effort approach.
-
-pip works on Unix/Linux, OS X, and Windows.
+pip works with CPython versions 2.6, 2.7, 3.3, 3.4, 3.5 and also pypy 
+which means pip works on the latest patch version of each of these minor 
+versions (i.e. 2.6.9 for 2.6, etc).Previous patch versions are supported 
+on a best effort approach. pip works on Unix/Linux, OS X, and Windows.
```

Comment:
-1 on contracting this into a single paragraphs, I think it detracts from the readability of skimming it for the information it wants to get across. And skimming through things is one way or another what a lot of people are going to try to do when gleaning technical information.